pkgs/top-level/all-packages.nix

```diff
@@ -27778,12 +27778,18 @@ with pkgs;
 
   qemu-utils = callPackage ../applications/virtualization/qemu/utils.nix {};
 
+  qgis-ltr-unwrapped = libsForQt5.callPackage ../applications/gis/qgis/unwrapped-ltr.nix {
+    withGrass = true;
+  };
+
   qgis-unwrapped = libsForQt5.callPackage ../applications/gis/qgis/unwrapped.nix {
-    withGrass = false;
+    withGrass = true;
```

Comment:
Doesn't `withGrass` already default to true here and in the case five lines above? Could be removed then.